pandas/core/frame.py

```diff
@@ -3392,7 +3392,7 @@ def pivot(self, index=None, columns=None, values=None):
         from pandas.core.reshape import pivot
         return pivot(self, index=index, columns=columns, values=values)
 
-    def stack(self, level=-1, dropna=True):
+    def stack(self, level=-1, dropna=True, sequentially=True):
```

Comment:
I don't quite understand. You want sequentially to default to ``None``? What would be the meaning?